deploy/base/goff-proxy/kustomization.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+generatorOptions:
```

Comment:
I've just been through removing this in #229 - disableNameSuffixHash - makes the configmap immutable.
That means we'd have to delete/re-create it to pick up changes which I don't think is the behaviour we want.

Reply:
ah, ok, yeah I'll remove it. It's just from copy/paste.